tests/README.md

```diff
@@ -1,5 +1,79 @@
-# Python Tests against the arkouda_server
+# Python Tests against the arkouda\_server
 
-  * depending on your setup you may need to set `PYTHONPATH` environment variable to find the `arkouda.py` module
-  * check the python source for runtime options
-  
\ No newline at end of file
+All Python tests can be run against a local or remote arkouda\_server. The arkouda project is configured so that
+no special Python paths need to be defined.
+  
+# The arkouda Python test harness
+
+The arkouda project has a [pytest](https://docs.pytest.org/en/latest/)-based Python test harness that can be executed 
+via the command-line interface (cli) or within the make file. 
+
+## pytest environment preparation
+
+There are two python libraries that must be installed to execute the arkouda test harness, pytest and pytest-env,
+both of which can be installed via pip:
+```
+pip install pytest, pytest-env
+```
+## Configuration of pytest: the pytest.ini file
+
+The pytest.ini file configures the arkouda Python test harness, specifically the following parameters:
+
+```
+[pytest]
+testpaths = 
+    tests/base_test.py
+    tests/client_test.py
+    tests/compare_test.py
+    tests/dtypes_tests.py
+    tests/groupby_test.py
+    tests/join_test.py
+    tests/operator_tests.py
+    tests/setops_test.py
+    tests/string_test.py
+    tests/where_test.py
+norecursedirs = .git dist build *egg* tests/deprecated/*
+python_functions = test*
+env =
+    ARKOUDA_SERVER_HOST=localhost
+    ARKOUDA_SERVER_PORT=5555
+    ARKOUDA_FULL_STACK_TEST=False
+    ARKOUDA_VERBOSE=False
+```
+* testpaths: shows the paths to all test files. For the time-being, the arkouda unit tests to be executed are 
+specified on a per-file basis, but pytest can also support directory-level configuration. 
+* norecursedirs: indicates which directories contain files that should be ignored, including tests/deprecated
+* python\_functions: the naming pattern for all test functions. Ih the case of arkouda, all python method (functions)
+  with names starting with "test" are executed by pytest
+* env: the pytest env variables needed to execute the arkouda Python test harness 
+
+## arkouda pytest environmental variables
```

Comment:
The server supports a `ARKOUDA_SERVER_CONNECTION_INFO` option that will write `host port` out to a file. I wonder if it would make sense to reuse that instead of having the host/port env vars.

Reply:
@ronawho possibly, but what if you want to vary the server you are working with?

Reply:
Yeah, that's a good point.